### ISSUE TITLE
Add support for Google Pay - TapAndPay library

### ISSUE THREAD

**Issue by kerrpeter** [xamarin/GooglePlayServicesComponents#196]
Any chance you are working an adding support for  Xamarin.Android bindings for the Google Pay, specifically the TapAndPay services for doing In-App provisioning, similar to the Xamarin.GooglePlayServices.Wallet package.

I've tried to manually create a binding for play-services-tapandpay-16.0.2.aar, but not having much luck with it

**Comment by Redth** [comment]
@kerrpeter do you have any links or information about this library?  I don't see it in Google's Maven Repo.

**Comment by kerrpeter** [comment]
It seems its not publicly available, but only available when you have white listed your email with google.
The Url is https://developers.google.com/pay/issuers/push-provisioning-api/
Perhaps I got email you the aar file I am trying to bind to in Xamarin

**Comment by moljac** [comment]
I did request whitelisting and will see (try to estimate effort).

Until then moving to low priority.

**Comment by kerrpeter** [comment]
@moljac Any update on this?

**Comment by moljac** [comment]
Google's answer to my request for whitelisting my email so I could legally get binaries and bind them:

>Hi Miljenko,
>
> Thank you for your email. I will consult with my team and get back to you soon.
>
> Sincerely,

**Comment by moljac** [comment]
@kerrpeter 

This is what I've found in my inbox:

> Hi Miljenko,

> Thank you for your patience. Unfortunately we cannot provide the SDK as it is a private SDK. If
> you have customers asking you to build bindings, you should be able to work with your customers
> to get access to the SDK through them

Seems like you'll have to provide binaries for me/us to bind them.

**Comment by kerrpeter** [comment]
Thanks Miljenko, 
Can you contact me at kerr_peter @ hotmail.com

**Comment by moljac** [comment]
@kerrpeter Ping sent

**Comment by xtremal** [comment]
@moljac, @kerrpeter 
Have you managed to build a binding library?
I am struggling with the same issue for days with no luck.

**Comment by kerrpeter** [comment]
Sorry, I no longer have access to this code from 4 years ago

**Comment by moljac** [comment]
@xtremal 

No. google never provided access to artifacts and/or code.

**Comment by xtremal** [comment]
Yeah, I know that Google keep it as "private". I mean have you managed to build a binding library or you never got the AAR? I thought that Peter sent you the tapandpay aar.

**Comment by moljac** [comment]
we never got aar

**Comment by kerrpeter** [comment]
If I recall I wasn't allowed to share it as it was private

**Comment by xtremal** [comment]
Yeah... absolute shame. 
However, thank you both and happy coding!
